### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by NGuldager**

### CODE REVIEW

**Review by daniellazarovlegacy — APPROVED** [review]

File: docs/config.json [hunk 1]
```diff
@@ -1,68 +1,110 @@
...
+            "quadrant": 2,
+            "ring": 0,
+            "label": "GCP Redis",
+            "active": true,
+            "moved": 0
+        },
+        {
+            "quadrant": 3,
+            "ring": 3,
+            "label": "GCP Cloud Tasks",
```
Comment (MadsRitter) [inline docs/config.json:98]: Why is Cloud tasks in Ring 3 (Hold)? Considering our use, and active migration to it, i would expect it to be in ring 0 (Adopt). We are however missing Bull, as a 3

Comment (NGuldager) [reply docs/config.json:98]: For me the key was the "new projects" in this description:
image

Would we want to use Cloud Tasks in new projects? Or find a solution using Pub/sub completely?

Comment (MadsRitter) [reply docs/config.json:105]: Cloud task is our queuing system that is being pushed heavily in production code. It is currently used in several different components, and we are actively moving more queue requirements to that function. So it is very much an adopt piece of tech :)

Comment (NGuldager) [reply docs/config.json:105]: Alright. Changed and added Bull 👍

**Review by MadsRitter — APPROVED** [review]

**Pull request merged**